content/docs/command-reference/import.md

```diff
@@ -86,6 +88,11 @@ repo at `url`) are not supported.
   current working directory). Directories specified in the path must already
   exist, otherwise this command will fail.
 
+- `--file <filename>` - specify a path and/or file name for the `.dvc` file
+  created by this command (e.g. `--file stages/stage.dvc`). This overrides the
```

Comment:
better to avoid using stage in the import context (for the future cc @jorgeorpinel )

Reply:
Agree. We have a pending check box about this in #608 epic. Also rel=#1366